static/js/script.js

```diff
@@ -26,19 +26,17 @@ if (flashes) {
 
 /* display profile social network field on click */
 const socialIcon = document.querySelector('.social-networks li')
-Array.from(document.socialIcon.forEach(li => {
-    if (socialIcon) {
-        socialIcon.addEventListener('click', (event) => {
-            event.preventDefault()
-        this.document.querySelector('label').classList.toggle('active')
-        })
-    }
+if (socialIcon) {
+    socialIcon.addEventListener('click', (event) => {
+        event.preventDefault()
+    this.document.querySelector('label').classList.toggle('active')
```

Comment:
`this` is useless here. `docoument` is global.

Reply:
even if I want `.active` class to be added to the `label` inside the clicked `li` ?